app/src/split/serial/central.c

```diff
@@ -0,0 +1,190 @@
+/*
+* Copyright (c) 2022 The ZMK Contributors
+*
+* SPDX-License-Identifier: MIT
+*/
+
+#include <zmk/split/common.h>
+#include <init.h>
+
+#include <sys/crc.h>
+#include <device.h>
+#include <drivers/uart.h>
+
+#include <logging/log.h>
+
+LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
+
+#include <zmk/event_manager.h>
+#include <zmk/events/position_state_changed.h>
+#include <zmk/matrix.h>
+
+#if DT_HAS_CHOSEN(zmk_split_serial)
```

Comment:
If we don't have a chosen, we should just `#error` instead of wrapping this all in a conditional.